src/annotation/annotations/require.js

```diff
@@ -1,156 +1,156 @@
-import { splitNamespace } from '../../utils';
-import uniq from 'lodash.uniq';
+import { splitNamespace } from '../../utils'
+import uniq from 'lodash.uniq'
 
-const reqRegEx = /^\s*(?:\{(.*)\})?\s*(?:(\$?[^\s]+))?\s*(?:-?\s*([^<$]*))?\s*(?:<?\s*(.*)\s*>)?$/;
+const reqRegEx = /^\s*(?:\{(.*)\})?\s*(?:(\$?[^\s]+))?\s*(?:-?\s*([^<$]*))?\s*(?:<?\s*(.*)\s*>)?$/
 
-export default function (env) {
+export default function require_ (env) {
```

Comment:
From what I’ve seen, we appended an underscore in case of a reserved word. I stuck with that. :)